test/modules/core/lib/effect-manager.spec.js

```diff
@@ -93,3 +98,38 @@ test('EffectManager#finalize', t => {
   t.deepEqual(resBegin, resEnd, 'Effect manager is finalized well');
   t.end();
 });
+
+test('EffectManager#setProps', t => {
+  const effect = new LightingEffect();
+  const effectManager = new EffectManager({gl, layerManager});
+  effectManager.setProps({effects: [effect]});
+
+  t.deepEqual(effectManager.effects, [effect], 'Effect manager setProps is ok');
+  t.equal(
+    effectManager.needApplyDefaultLighting,
+    false,
+    'Effect Manager needApplyDefaultLighting is ok'
```

Comment:
Use more descriptive language, e.g. `EffectManager should not need to apply default lighting`